src/core/concepts/getting-started/data-pipelines.md

```diff
@@ -4,7 +4,7 @@ DataJoint is a free open-source framework for creating scientific data pipelines
 directly from Python or MATLAB (or both). The data are stored in a language-independent
 way, accessible via both Python and MATLAB. In DataJoint, a data pipeline is a sequence
 of steps (see also [DAGs](./diagrams))) with integrated data storage at each step, as defined
-by the [tables](../reproduce/tabletiers). 
+by the [tables](../reproduce/table-tiers.md). 
```

Comment:
the current link goes to `getting-started/reproduce/tabletiers/`, so I think we need one more layer up
```suggestion
by the [tables](../../reproduce/table-tiers.md). 
```